microprofile.ls/org.eclipse.lsp4mp.ls/src/test/java/org/eclipse/lsp4mp/services/ApplicationPropertiesHoverTest.java

```diff
@@ -222,8 +219,7 @@ public void hoverResolveTwoSteps() throws BadLocationException {
 		String value = "property.one = hello\n" + //
 				"property.two = ${property.one}\n" + //
 				"property.three = ${property.two}\n" + //
-				"property.four = ${property.three}\n" +
-				"property.five = ${property|.four}";
+				"property.four = ${property.three}\n" + "property.five = ${property|.four}";
```

Comment:
This got autoformatted to one line instead of two

Reply:
fixed